torch/jit/__init__.py

```diff
@@ -2081,17 +2081,22 @@ def _check_directly_compile_overloaded(obj):
 Error.__name__ = "Error"
 Error.__qualname__ = "Error"
 
-def _get_named_tuple_properties(obj):
+def _get_named_tuple_properties(obj, need_types=True):
```

Comment:
Booleans that change the return type of a function are not a great way to organize code. Can these be made into two functions.